### DISCUSSION TITLE
Achievements on Non-Experimental Version?

### DISCUSSION THREAD

**Discussion by TideGear** [Detanup01/gbe_fork#124] [Q&A] [answered]
Do achievements still pop up on the regular, non-experimental version? If so, where can you see what achievements you've gotten without the overlay?

**Comment by DogancanYr** [comment, marked as answer]
It is impossible because there is no interface in the regular version.
All achievements are saved in this location to be shown by the interface.
https://github.com/Detanup01/gbe_fork/blob/dev/post_build/README.release.md#savessettings-location
Or you can use the achievement watcher.
Since the application is old, you need to show the file with the achievement information from the settings.
To turn off achievement notifications, you can turn it off by looking at the default settings folder.
